x/cdp/abci.go

```diff
@@ -21,6 +21,19 @@ func BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock, k Keeper) {
 			k.HandleNewDebt(ctx, cp.Denom, dp.Denom, timeElapsed)
 		}
 
+		// call our update fees method for the risky cdps
+		e := k.UpdateFeesForRiskyCdps(ctx, cp.Denom)
```

Comment:
```suggestion
		err := k.UpdateFeesForRiskyCdps(ctx, cp.Denom)
```

nit: stick to convention of the rest of the codebase and call error results `err`